feat(UI): accent colours

unseen-ninja opened the pull request:
This PR simply adds support for personalised accent colours as seen in other userstyles.
Instead of always using `green` for the `--color-brand-green` users can now chose from all available Catppuccin colours. c:

<img width="273" alt="Screenshot 2023-03-23 at 11 31 50" src="https://user-images.githubusercontent.com/35840154/227176574-1e25d5aa-ae19-4335-a007-77142bdb5077.png">

Example with `mauve` as accent colour:
<img width="172" alt="Screenshot 2023-03-23 at 11 34 09" src="https://user-images.githubusercontent.com/35840154/227177120-b8ed4944-0bfc-4aad-9748-0c8ed3b9c223.png"> <img width="114" alt="Screenshot 2023-03-23 at 11 34 20" src="https://user-images.githubusercontent.com/35840154/227177160-58a5af4b-33fb-4e3b-ba87-2a4b4ea9528e.png"> <img width="216" alt="Screenshot 2023-03-23 at 11 34 30" src="https://user-images.githubusercontent.com/35840154/227177200-87c860a7-763d-4361-a87e-1be24acc6f43.png">

unseen-ninja committed:
feat(UI): accent colours

andreasgrafen commented:
ah … Auto formatting also removed indents from empty lines as it seems. I'll pretend that this is intended and mark it as a feature. :^)

unseen-ninja committed:
chore: bump version number

Converting the version number to follow semantic versioning as well as bumping it to 1.1.0

thismoon commented:
Thanks!
